tests/test_tensorflow_sparse_data_api.py

```diff
@@ -231,3 +259,71 @@ def test_except_with_multiple_nz_value_record_of_batch_x_y_rows(
                 x_array=x, y_array=y, batch_size=BATCH_SIZE
             )
             model.fit(tiledb_dataset, verbose=0, epochs=2)
+
+    def test_generator_sparse_sparse(self, tmpdir, model):
```

Comment:
Same here for naming.

Reply:
The are just the internal functions of the class just adding a test prefix.